sync/sync.go

```diff
@@ -133,7 +150,9 @@ func (s *Synchronizer) fetchUnknownClasses(ctx context.Context, stateUpdate *cor
 func (s *Synchronizer) verifierTask(ctx context.Context, block *core.Block, stateUpdate *core.StateUpdate,
 	newClasses map[felt.Felt]core.Class, resetStreams context.CancelFunc,
 ) stream.Callback {
+	timer := prometheus.NewTimer(s.opTimers.WithLabelValues("verify"))
```

Comment:
lets add the labels as `const`, this applies to `store` and `fetch`

Reply:
I dont tend to do that when the constant is used only once. Since it is just an indirection that serves no purpose IMO. It is up to you, I don't mind changing it.

Reply:
I think we should extract them out to a `const`.